### ISSUE TITLE
[native] Use HiveQueryRunner in presto-native-execution/presto_cpp/main/tests/TaskManagerTest.cpp

### ISSUE THREAD

**Issue by majetideepak** [prestodb/presto#21442]
We currently use DuckDB to validate the results of the test.
Using the HiveQueryRunner will help avoid a dependency.

**Comment by majetideepak** [comment]
CC: @duanmeng

**Comment by duanmeng** [comment]
> We can probably use JNI here to invoke the HiveQueryRunner.

@majetideepak Yes, I think so. It would be nice to be able to launch an embedded HiveQueryRunner on the fly during the unit/fuzzer test.  Masha added the PrestoQueryRunner in the Presto Native, and Krishna is actively working on integrating this with the Aggregation fuzzer in Velox. cc @mbasmanova @kgpai
